tests/test_state_metrics/test_bures_distance.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for bures_distance."""
```

Comment:
I'm curious if there are any specific examples in the literature, academic course notes, etc. where the Bures distance is calculated between two different specific density matrices with a known result. That might be nice to add as a test case for a sanity check. One could also look at the unit tests for say the `qutip` package that also has a function for the `bures_distance`.

Reply:
I checked  the test in qutip but they don't have any specific test for the `bures_dist`. They only check an inequality, here is the test:  [https://github.com/qutip/qutip/blob/master/qutip/tests/test_metrics.py](https://github.com/qutip/qutip/blob/master/qutip/tests/test_metrics.py).

I have added an example for pure states shown in this book: [Elements of Quantum Computation and Quantum Communication](https://books.google.com.mx/books/about/Elements_of_Quantum_Computation_and_Quan.html?id=cEPSBQAAQBAJ&source=kp_book_description&redir_esc=y), the example is the 3.42 in the book.  I couldn't find more examples but I will keep searching.